vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputProps.h

```diff
@@ -84,6 +84,22 @@ static inline void fromRawValue(
   }
 }
 
+static inline void
+fromRawValue(const PropsParserContext &context, const RawValue &value, facebook::react::TextAlignment &textAlignment) {
```

Comment:
We should include this instead of creating another definition:
#include <react\renderer\attributedstring\conversions.h>

Reply:
This is great feedback. Thank you!